packages/@dcl/inspector/src/lib/logic/in-memory-storage.ts

```diff
@@ -28,27 +28,40 @@ export function createInMemoryStorage(initialFs: Record<string, Buffer> = {}) {
   }
 }
 
+function normalizePath(path: string) {
+  return path.replace(/^\/+/g, '')
```

Comment:
add a comment or an example about this so i dont have to understand the regexp 🙏🏼